extension/css/cryptup.css

```diff
@@ -2326,9 +2326,11 @@ div#reply_message_successful_container > table div.replied_from {
   margin-bottom: 4px;
 }
 
-div#reply_message_successful_container > table div.replied_to {
+div#reply_message_successful_container > table div.replied_to,
+table div.replied_cc,
```

Comment:
```diff
- table div.replied_cc
+ div#reply_message_successful_container > table div.replied_cc
```

Reply:
I have done changing the incorrect CSS declaration here, thanks!